cfgov/unprocessed/js/shims/es5-shim.js

```diff
@@ -0,0 +1,11 @@
+/* ==========================================================================
+   ES 5 Shim/Sham, added for BB7.
+   ========================================================================== */
+
+'use strict';
+
+if( window.Modernizr && window.Modernizr.es5 === false ) {
+  // Global modules.
+  require( 'es5-shim' ); // eslint-disable-line global-require
+  require( 'es5-sham' ); // eslint-disable-line global-require
```

Comment:
yeah, I think it is fine for our case; the only methods that would silently fail are newer Object methods if you passed a an erroneous type.

https://github.com/es-shims/es5-shim/blob/master/es5-sham.js#L460